packages/web-react/src/components/UNSTABLE_Slider/stories/UNSTABLE_Slider.stories.tsx

```diff
@@ -0,0 +1,149 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React, { useState } from 'react';
+import { ValidationStates } from '../../../constants';
+import { SpiritSliderProps } from '../../../types';
+import ReadMe from '../README.md';
+import { UNSTABLE_Slider } from '..';
+
+const meta: Meta<typeof UNSTABLE_Slider> = {
+  title: 'Components/UNSTABLE_Slider',
+  component: UNSTABLE_Slider,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+    layout: 'centered',
+  },
+  argTypes: {
+    helperText: {
+      control: 'text',
+      table: {
+        defaultValue: { summary: undefined },
+      },
+    },
+    id: {
+      control: 'text',
+      table: {
+        defaultValue: { summary: undefined },
+      },
+    },
+    isFluid: {
+      control: 'boolean',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    isLabelHidden: {
+      control: 'boolean',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    isRequired: {
+      control: 'boolean',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    isSelected: {
+      control: 'boolean',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    label: {
+      control: 'text',
+      table: {
+        defaultValue: { summary: undefined },
+      },
+    },
+    max: {
+      control: 'number',
+      table: {
+        defaultValue: { summary: 100 },
+      },
+    },
+    min: {
+      control: 'number',
+      table: {
+        defaultValue: { summary: 0 },
+      },
+    },
+    step: {
+      control: 'number',
+      table: {
+        defaultValue: { summary: 1 },
```

Comment:
Shouldn't we reuse the default values defined in types?

Reply:
imported from constants 👍